information/gmt_data_server.txt

```diff
@@ -81,3 +81,33 @@
 /server/earth/earth_mask/	earth_mask_01m_p.grd	01m	p	1	0	998K	0		2020-06-01		-					-			Earth Mask at 1x1 arc minutes (GSHHG features < 3 km^2 in area are skipped)
 /server/earth/earth_mask/	earth_mask_30s_p.grd	30s	p	1	0	2.6M	0		2020-06-01		-					-			Earth Mask at 30x30 arc seconds (GSHHG features < 1 km^2 in area are skipped)
 /server/earth/earth_mask/	earth_mask_15s_p.grd	15s	p	1	0	7.5M	0		2020-06-01		-					-			Earth Mask at 15x15 arc seconds (all GSHHG features included)
+#
+# Earth Day images
+#
+/server/earth/earth_day/	earth_day_01d_p.grd		01d	p	1	0	 69K	0		2020-06-01		-					-			Earth Daytime Image at 1x1 arc degrees [NASA imagery - Blue Marble]
```

Comment:
`earth_day_01d_p.grd` or `earth_day_01d_p.tiff`?

BTW, I can't read the earth_day and earth_night directories on the gmt server.
```
-bash-4.2$ ll
total 8
drwx--x--x  2 pwessel gmt  306 Jun  6 12:25 earth_day
drwxr-xr-x  2 pwessel gmt 4096 Jun  6 09:56 earth_mask
drwx--x--x  2 pwessel gmt  330 Jun  6 12:31 earth_night
drwxr-xr-x 17 pwessel gmt 4096 Jun  1 18:53 earth_relief
-bash-4.2$ ls earth_day/*
ls: cannot access earth_day/*: No such file or directory
```

Reply:
Whoops. OK fixed. chmod had to wait until all files downloaded. now done